<launch>
    <node name="pub_dt2_bndpts" pkg="sitl_dvrk" type="pub_dt2_masks.py" output="screen">
        <param name="calib_dir"           type="str"    value="L2R"/>
        <param name="frame_id"            type="str"    value="ecm_opencv"/>
        <param name="obj_type"            type="str"    value="liver"/>
        <param name="seg_model_path"      type="str"    value="/home/$(env USER)/dt2_dataset/liver/results_noicra/model_final.pth"/>
        <param name="seg_score_thresh"    type="double" value="0.5"/>
        <param name="num_goals"           type="int"    value="8"/>
        <param name="cnt_area_thr"        type="int"    value="20000"/>
        <param name="tip_child_frame_id"  type="str"    value="dt2_inst_tip"/>
        <param name="jaw_child_frame_id"  type="str"    value="dt2_inst_jaw"/>
        <param name="psm1_inst"           type="str"    value="PCH"/>
        <param name="psm2_inst"           type="str"    value=""/>
        <param name="kpt_model_path"      type="str"    value="/home/$(env USER)/dt2_dataset/kpts/results_t3/model_final.pth"/>
        <param name="kpt_score_thresh"    type="double" value="0.7"/>
    </node>

    <?ignore
    
    ?>
</launch>